src/orca/services/nextflowtower/ops.py

```diff
@@ -1,8 +1,10 @@
+from dataclasses import field
 from functools import cached_property
 from typing import ClassVar, Optional
 
 from pydantic.dataclasses import dataclass
 
+from orca import logger
```

Comment:
Hmm.. @BWMac and I were discussing the logging module, and I've always read that the best practice is to have instantiate a logger per module: https://stackoverflow.com/questions/15727420/using-logging-in-multiple-modules.

Thoughts?

Reply:
Mm, good catch! I'll work on updating this.